test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -1779,6 +1780,9 @@ protected static boolean isXPackTemplate(String name) {
         if (name.startsWith(".fleet-")) {
             return true;
         }
+        if (name.startsWith("behavioral_analytics-")) {
```

Comment:
ℹ️ All "default" templates (and component templates) are skipped during the clean up phase. Adding our is preventing some failure.